composables/loading.js

```diff
@@ -0,0 +1,16 @@
+export const useLoading = () =>
+  useState('loading', () => ({
+    loading: false,
+  }))
```

Comment:
From the discussion on Discord: https://discord.com/channels/734077874708938864/1045190420440821821/1069745483741941880. This probably would be better if it was just `useState('loading', () => false)`. Then value returned by `useState` can be watched easier — it would be ref to a boolean instead of a ref to reactive object, so you won't need to dive deep to subscribe to updates of `loading` (`watch(() => loading.value.loading, …)` vs `watch(loading, …)`). Also easier to update in functions below, just `useLoading().value = <true/false>`.

Reply:
will do this, thanks